generator/web/src/main/kotlin/me/kcra/takenaka/generator/web/pages/classPage.kt

```diff
@@ -167,45 +167,101 @@ fun GenerationContext.classPage(klass: MappingTreeView.ClassMappingView, hash: S
                     fieldMask = fieldMask and Modifier.PUBLIC.inv() and Modifier.STATIC.inv() and Modifier.FINAL.inv()
                 }
 
-                addContentSpacer()
-                h4 {
-                    +"Field summary"
-                }
-                table(classes = "styled-table styled-mobile-table") {
-                    thead {
-                        tr {
-                            th {
-                                +"Modifier and Type"
+                var fields = klass.fields
+
+                if (classTypeOf(klassDeclaration.modifiers) == ClassType.ENUM) {
+                    val enumModifier = Modifier.PUBLIC or Modifier.STATIC or Modifier.FINAL or Opcodes.ACC_ENUM
+
+                    val (enumFields, nonEnumFields) = fields.partition { (it.modifiers and enumModifier) == enumModifier && it.srcDesc == 'L' + klass.srcName + ';' }
```

Comment:
this check seems a bit overkill, just checking for the existence of `ACC_ENUM` on the field modifiers should be enough

also, filtering collections isn't that expensive, so you can use the same approach as methods-constructors